devtools/x/src/lint/whitespace.rs

```diff
@@ -80,9 +81,22 @@ impl ContentLinter for TrailingWhitespace {
     }
 }
 
-fn skip_whitespace_checks(ext: Option<&str>) -> RunStatus {
-    match ext {
-        Some("exp") => RunStatus::Skipped(SkipReason::UnsupportedExtension(ext)),
-        _ => RunStatus::Executed,
+fn skip_whitespace_checks<'l>(file: &FileContext<'l>) -> RunStatus<'l> {
+    // glob based opt outs
+    let patterns = [
+        ".github/actions/*/dist/*",
+    ];
+
+    if let Some(pattern) = patterns.iter().find(|s| Pattern::new(s).unwrap().matches_path(file.file_path())) {
+        return RunStatus::Skipped(SkipReason::GlobExemption(pattern));
+    };
+
+
+    // extension based opt outs
+    match file.extension() {
+        Some("exp") => return RunStatus::Skipped(SkipReason::UnsupportedExtension(file.extension())),
+        _ => (),
     }
+
+    RunStatus::Executed
```

Comment:
Looks good! At some point (not in this PR of course) we should move these so they're configurable in the x.toml file but this is sufficient for now.